finance/community-updates/2020-10.txt

```diff
@@ -0,0 +1,68 @@
+Dear Astropy Community,
+
+Once again, the Interim Finance Committee would like to give you some updates
+on our activities over the couple of months.
+
+- The process for selecting Inclusion, Diversity, and Equity in Astropy
+  proposals has been completed [1]. We were delighted with the high-quality of the
+  proposals received, and with the community discussion which resulted. Thanks
+  to everybody who took part!
+
+  Based on this process, two proposals have been funded: one to fund an
+  Astropy intern through Outreachy [2] and the other to develop a peer
+  mentoring network for women of color and other marginalized groups [3].
+  We're really excited to see these projects make progress!
```

Comment:
Two other proposals which were focused on conference attendance were funded from the travel budget rather than the IDEA budget.